.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
@keunes What indent do we want to use? Most files use 2 spaces, some files use 4 spaces. We should make this a deliberate decision.

Reply:
Personal preference? 2.